clients/java/src/main/java/io/camunda/zeebe/client/impl/response/DecisionImpl.java

```diff
@@ -40,6 +40,34 @@ public DecisionImpl(final DecisionMetadata metadata) {
         metadata.getTenantId());
   }
 
+  /**
+   * A constructor that provides an instance with an empty String for a tenantId.
+   *
+   * <p>From version 8.3.0, the java client supports multi-tenancy for this command, which requires
+   * the <code>tenantId</code> property to be defined. This constructor is only intended for
+   * backwards compatibility in tests.
+   *
+   * @deprecated since 8.3.0, use {@link DecisionImpl#DecisionImpl(String dmnDecisionId, String
+   *     dmnDecisionName, int version, long decisionKey, String dmnDecisionRequirementsId, long
+   *     decisionRequirementsKey, String tenantId)}
+   */
+  public DecisionImpl(
+      final String dmnDecisionId,
+      final String dmnDecisionName,
+      final int version,
+      final long decisionKey,
+      final String dmnDecisionRequirementsId,
+      final long decisionRequirementsKey) {
+    this(
+        dmnDecisionId,
+        dmnDecisionName,
+        version,
+        decisionKey,
+        dmnDecisionRequirementsId,
+        decisionRequirementsKey,
+        "");
```

Comment:
❓ WDYT about putting the default tenant in here instead of an empty string? I feel like this matches the actual behaviour of Zeebe more closely. If no tenant is provided, we use the default.

Reply:
Same comment goes for the other places.

Reply:
Yup, a similar discussion [here](https://github.com/camunda/zeebe/pull/14107#discussion_r1315641044).

I'll do it like that then.

This will also trigger a change in the validation on the Gateway (see my comment in the link above).